### DISCUSSION TITLE
IMX500 Sony AI Accelerator Ultralytics model export = imx Convertion error

### DISCUSSION THREAD

**Discussion by ARCHANGELOVE** [ultralytics/ultralytics#17755] [Q&A]
Hi everyone im facing error on convertion YOLOv8n-obb model to imx. Is this a bug

this is the command i run

yolo export model=yolov8n-obb.pt format=imx

File "/home/ubuntu/.pyenv/versions/3.9.20/lib/python3.9/site-packages/ultralytics/nn/modules/head.py", line 124, in _inference
dbox = self.decode_bboxes(
TypeError: decode_bboxes() got an unexpected keyword argument 'xywh'

**Comment by UltralyticsAssistant** [comment]
👋 Hello @ARCHANGELOVE, thank you for sharing your issue with Ultralytics 🚀! We recommend visiting our [Docs](https://docs.ultralytics.com) for a wealth of information including [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples, where you might find answers to common questions.

It seems you might be encountering a 🐛 bug. To assist our debugging process, could you please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/)?

Moreover, if this relates to a custom training ❓ question, please share additional details like dataset samples and training logs, and ensure you're aligning with our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

Engage with our Ultralytics community where it best suits your needs. Join the conversation on [Discord](https://ultralytics.com/discord) 🎧 for real-time chat, explore in-depth discussions on [Discourse](https://community.ultralytics.com), or dive into our [Subreddit](https://reddit.com/r/ultralytics) to share and gain knowledge.

## Upgrade

To check if your issue has been resolved, consider upgrading to the latest `ultralytics` package along with all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/):

```bash
pip install -U ultralytics
```

## Environments

YOLO can be run on any of the following updated environments (ensure all dependencies including [CUDA](https://developer.nvidia.com/cuda), [CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/), and [PyTorch](https://pytorch.org/) are preinstalled):

- **Notebooks** with available free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM via our [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI through our [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image** option described in our [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

If the above badge is green, it indicates that all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule) tests are currently passing. These CI tests validate the correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) across macOS, Windows, and Ubuntu every 24 hours and upon each commit.

An Ultralytics engineer will be on hand soon to assist you further! 😊

**Comment by glenn-jocher** [comment]
@ARCHANGELOVE thank you for reaching out. The error suggests that the `decode_bboxes` function encountered an incompatible argument during export. Currently, IMX format export may not fully support the YOLOv8n-obb model. Please refer to our documentation (https://docs.ultralytics.com/integrations) for supported formats and updates, or consider reaching out to Sony's IMX support for compatibility details.
